causal-inference-for-the-brave-and-true/03-Stats-Review-The-Most-Dangerous-Equation.ipynb

```diff
@@ -623,7 +623,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "With this at hand, we can say that we are 95% confident that the true difference between the online and face to face group falls between -8.37 and -1.44. We can also construct a **z statistic** by dividing the difference in mean by the \\\\(SE\\\\) of the differences. \n",
+    "With this at hand, we can say that the true difference between the online and face to face groups falls between -8.37 and -1.44 95% of the time. We can also construct a **z statistic** by dividing the difference in mean by the \\\\(SE\\\\) of the differences. \n",
```

Comment:
Thats actually not true. This is a common misconception when interpreting a confidence interval. Since in frequentist statistics we see the parameters as fixed, unknown values, the probability that the true difference is in the CI is either 0 (if it doesn't ) or 1 if (it does). The 95% is then a confidence that along multiple experiments, 95% of the 95 CI will contain the true difference. Its a confidence about the algorithm, not about this individual CI